Mathlib/Algebra/Module/FinitePresentation.lean

```diff
@@ -315,4 +315,13 @@ lemma Module.FinitePresentation.isLocalizedModule_map
     ext x
     simpa using LinearMap.congr_fun e (f x)
 
+instance Module.FinitePresentation.isLocalizedModule_mapExtendScalars
+    {M' : Type*} [AddCommGroup M'] [Module R M'] (f : M →ₗ[R] M') [IsLocalizedModule S f]
+    {N' : Type*} [AddCommGroup N'] [Module R N'] (g : N →ₗ[R] N') [IsLocalizedModule S g]
+    (Rₛ) [CommRing Rₛ] [Algebra R Rₛ] [Module Rₛ M'] [Module Rₛ N']
+    [IsScalarTower R Rₛ M'] [IsScalarTower R Rₛ N'] [IsLocalization S Rₛ]
+    [Module.FinitePresentation R M] :
+      IsLocalizedModule S (IsLocalizedModule.mapExtendScalars S f g Rₛ) :=
+  IsLocalizedModule.of_linearEquiv _ _ _
```

Comment:
This declaration shares most of its variables with the one above, can you make them into one section?